go/cli/mcap/utils/utils.go

```diff
@@ -111,10 +115,152 @@ func FormatTable(w io.Writer, rows [][]string) {
 	tw.Render()
 }
 
+func inferWriterOptions(info *mcap.Info) *mcap.WriterOptions {
+	// assume if there are no chunk indexes, the file is not chunked
```

Comment:
Can this assumption fail on an unindexed file?

Reply:
if the file is chunked but not indexed, yeah it'll fail. I think though the only way to guard against that would be to scan the entire file to see if a chunk is present. I'm not sure that's worth it, but it's something we could do.

This is where it would be handy to have the writer options in the header.